### DISCUSSION TITLE
https://prototypefund.de a funding possibility?

### DISCUSSION THREAD

**Discussion by JohnFitz** [cboxdoerfer/fsearch#429] [General]
Hi Christian, I was watching the recent [NextCloud](https://youtu.be/Uc3gb-rY7rI) conference videos, and they mentioned the [Protoype Fund](https://prototypefund.de/) which they've used to fund some of their new features. I just thought it might be of interest to since your Impressum says you live in Germany and thought perhaps if there's a feature you'd like to try to build but needed more time to do it this could be an avenue. 

Maybe you're already aware of the fund, but if you're not, it's open to applications from German citizens and it's supposed to be a relatively simple application process. They fund a wide assortment of OSS. 

I hope I'm not overstepping a boundary here, but selfishly, I find FSearch so useful that I'd love to see your project funded in a meaningful way :grin:  

Anyway, just thought I'd put that out there. Feel free to ignore.

**Comment by cboxdoerfer** [comment]
Thanks, I haven't heard of that project. I'll surely have a look at it, but to be honest I kind of doubt that a small and niche project like FSearch would qualify for such funding, even though it would be really great.
